weather.py

```diff
@@ -1,104 +1,126 @@
 import csv
 from datetime import datetime
 
-DEGREE_SYBMOL = u"\N{DEGREE SIGN}C"
+DEGREE_SYMBOL = u"\N{DEGREE SIGN}C"
 
 
 def format_temperature(temp):
-    """Takes a temperature and returns it in string format with the degrees
-        and celcius symbols.
+    return (f"{temp}{DEGREE_SYMBOL}")
 
-    Args:
-        temp: A string representing a temperature.
-    Returns:
-        A string contain the temperature and "degrees celcius."
-    """
-    return f"{temp}{DEGREE_SYBMOL}"
+def convert_date(iso_string):
+    date = datetime.fromisoformat(iso_string)
+    return(date.strftime("%A %d %B %Y"))
+    
+    
 
+def convert_f_to_c(temp_in_farenheit):
+    temp_in_celcius = round((float(temp_in_farenheit) - 32) * 5/9, 1)
+    return temp_in_celcius
 
-def convert_date(iso_string):
-    """Converts and ISO formatted date into a human readable format.
 
-    Args:
-        iso_string: An ISO date string..
-    Returns:
-        A date formatted like: Weekday Date Month Year e.g. Tuesday 06 July 2021
-    """
-    pass
+def calculate_mean(weather_data):
+    total = 0
+    for data in weather_data:
+        total = total + float(data)
+    return total/len(weather_data)
+    
+
+def load_data_from_csv(file):
+    weather_data = []
+    with open(file) as csv_file:
+        results = csv.reader(csv_file)
+        count = 0
+        for line in results:
+            if line == []:
+                pass
+            
+            elif count != 0:
+                updated_line = line
+                updated_line[1]= int(line[1])
+                updated_line[2]= int(line[2])
+                weather_data.append(updated_line)
```

Comment:
To improve code readability, this could also be an `else` statement. In this way, the reader is essentially only checking one condition (`line == []` or not), rather than 2 (`line == []` or not, and then `count != 0` or not)